app/views/admin/partners/_form.html.erb

```diff
@@ -22,6 +22,22 @@
   <br>
   <hr>
 
+  <h2>Service Area</h2>
+
+  <div class="partner__neighbourhoods">
+    <% @all_neighbourhoods.each do |neighbourhood| %>
+      <%= fields_for "partner[service_areas_attributes][#{neighbourhood.id}]" do |sna|%>
```

Comment:
So I think? This is a place where simple_form's association dialogue would be appropriate

Reply:
`|sna| %>` missing space here is killing me lmao